easybuild/easyconfigs/w/wcslib/wcslib-7.11-GCC-11.2.0.eb

```
@@ -0,0 +1,41 @@
+# Contribution from Imperial College London (UK)
+# Uploaded by J. Sassmannshausen
+
+easyblock = 'ConfigureMake'
+
+name = 'wcslib'
+version = '7.11'
+
+homepage = 'https://www.atnf.csiro.au/people/mcalabre/WCS/'
+description = """The FITS "World Coordinate System" (WCS) standard defines keywords 
+and usage that provide for the description of astronomical coordinate systems in a 
+FITS image header."""
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['ftp://ftp.atnf.csiro.au/pub/software/wcslib/']
+sources = ['%(name)s-%(version)s.tar.bz2']
+checksums = ['46befbfdf50cd4953896676a7d570094dc7661e2ae9677b092e7fb13cee3da5f']
+
+builddependencies = [
+    ('binutils', '2.37'),
+    ('M4', '1.4.19'),
+    ('flex', '2.6.4'),
+]
+
+dependencies = [
+    ('CFITSIO', '3.49'),
+    ('pgplot', '5.2.2'),
+    ('X11', '20210802'),
+]
+
+configure_cmd = './configure --with-cfitsiolib=$EBROOTCFITSIO/lib --with-cfitsioinc=$EBROOTCFITSIO/include '
+configure_cmd += '--with-pgplotlib=$EBROOTPGPLOT/lib --with-pgplotinc=$EBROOTPGPLOT/include '
```

configure options should go in `configopts`, no need to change the `configure_cmd` itself
```suggestion
configopts = '--with-cfitsiolib=$EBROOTCFITSIO/lib --with-cfitsioinc=$EBROOTCFITSIO/include '
configopts += '--with-pgplotlib=$EBROOTPGPLOT/lib --with-pgplotinc=$EBROOTPGPLOT/include '
```

---

Thanks. I knew there was a better way of doing this but at the time I could not see the wood of the forest.